modelerfour/modeler/interpretations.ts

```diff
@@ -71,6 +71,79 @@ export function getValidEnumValueName(originalString: string): string {
 }
 
 export class Interpretations {
+  isTrue(value: any) {
+    return (value === true || value === 'true' || value === 'True' || value === 'TRUE');
+  }
+
+  getConstantValue(schema: OpenAPI.Schema, value: any) {
+
+    switch (schema.type) {
+      case JsonType.String:
+        switch (schema.format) {
+          // member should be byte array
+          // on wire format should be base64url
+          case StringFormat.Base64Url:
+            // return this.parseBase64UrlValue(value);
```

Comment:
Are the commented-out returns just there for explanation or should they be used?

Reply:
I'm waffling on if I want to parse the value. I passed it straight thru for now, but I'm thinking about going back.

Reply:
Sounds fine, might leave a comment to explain why those are still there just so we remember in the future